### PR TITLE
Update changelog and readme

### PR DESCRIPTION
**Pull request by russelarms**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?

## Description

Update changelog and readme.

## Related PRs or issues

[AB#77375](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/77375)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +17,10 @@
 
 * **[Fix]** Fix converting string with the Unicode symbols (for example in push notification).
 
+#### UWP
+
+* **[Feature]** App Center SDK supports **arm64** on UWP.
```
Comment (annakocheshkova) [inline CHANGELOG.md]: Maybe it's better to use the sentence from DotNet:
> [Feature] Support ARM64 architecture.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,19 @@
 
 # Introduction
-The App Center Unity SDK enables you to use App Center Analytics, Crash and Distribution within your Unity games. 
+The App Center Unity SDK enables you to use App Center Analytics, Crash and Distribution within your Unity games.
```
Comment (guperrot) [inline README.md:4]:
```suggestion

The App Center Unity SDK enables you to use App Center Analytics, Crash and Distribution within your Unity games.
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,19 @@
...
 ## 1. Prerequisites
 
-Before you begin, make sure that your project is set up in Unity 5.6 or later.
+Before you begin, make sure that your project is set up in Unity 2018.1 or later.
 
 The App Center SDK for Unity supports the following platforms:
 
 * iOS (9.0 or later)
 * Android (5.0/API 21 or later)
 * UWP (Build 10240 or later)
```
Comment (MatkovIvan) [inline README.md]: 16299

File: CHANGELOG.md [hunk 4]
```diff
@@ -13,6 +15,10 @@
 
 * **[Fix]** Fix converting string with the Unicode symbols (for example in push notification).
 
+#### UWP
```
Comment (MatkovIvan) [inline CHANGELOG.md:18]: We dropping support for .NET scripting backend

File: CHANGELOG.md [hunk 5]
```diff
@@ -13,6 +15,10 @@
...
+
+* **[Feature]** App Center SDK supports **arm64** on UWP.
```
Comment (MatkovIvan) [inline CHANGELOG.md]: ARM64 only supported by Unity 2019.x or above

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -205,7 +212,7 @@ Updated native SDK versions:
 * iOS from `1.13.2` to [1.14.0](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/1.14.0)
 * UWP from `1.13.2` to [1.14.0](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/1.14.0)
 
-* **[Fix]** Fixed `async`/`await` operators support by SDK’s methods for .NETStandard 2.0 profile
+* **[Fix]** Fixed `async`/`await` operators support by SDK's methods for .NETStandard 2.0 profile
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fixed `async`/`await` operators support by SDK's methods for .NETStandard 2.0 profile.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -205,7 +212,7 @@ Updated native SDK versions:
...
 * **[Fix]** Fixed iOS application crash when trying to pass exception without stack trace to `Crashes.TrackError`
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fixed iOS application crash when trying to pass exception without stack trace to `Crashes.TrackError`.
```

**Review by guperrot — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**